moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -131,16 +134,7 @@ export default function MapView({ projectName, projectStatuses }) {
     // close the map projectFeature map popup
     setClickedProjectFeature(null);
     // move the map
-    mapRef.current?.fitBounds(bbox(featureCollection), {
-      maxZoom: 19,
-      // accounting for fixed top bar
-      padding: {
-        top: 75,
-        bottom: 75,
-        left: 75,
-        right: 75,
-      },
-    });
+    mapRef.current?.fitBounds(bbox(featureCollection), fitBoundsOptions);
```

Comment:
thanks for applying the same options here as well!

Reply:
All good, I'll expand that config with keys for each case! 🙏